src/khoj/routers/api_chat.py

```diff
@@ -128,6 +133,60 @@ def chat_history(
     return {"status": "ok", "response": meta_log}
 
 
+@api_chat.get("/public/history")
```

Comment:
Maybe we can call this `/public/chat`? It'll generalize better once we allow for user created public agents and automations